src/main/webapp/resources/css/inward_five_five.css

```
@@ -0,0 +1,365 @@
+@media print{
+    .posbill{
+        position: relative!important;
+        margin-top: -2cm!important;
+        width: 100%;
+        padding: 0px!important;
+        margin: 0px!important;
```

Stylelint (17.14.0)

[error] Overridden property "margin-top" by shorthand "margin" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)